src/ContactList.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import Contact from './Contact';
+
+/* eslint max-len: [1, {"ignoreUrls": true}] */
+class ContactList extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      contacts: [
+        {
+          _id: 1,
+          name: 'Dale Cooper',
+          occupation: 'FBI Agent',
+          avatar: 'https://upload.wikimedia.org/wikipedia/en/5/50/Agentdalecooper.jpg'
+        },
+        {
+          _id: 2,
+          name: 'Spike Spiegel',
+          occupation: 'Bounty Hunter',
+          avatar: 'http://vignette4.wikia.nocookie.net/deadliestfiction/images/d/de/Spike_Spiegel_by_aleztron.jpg/revision/latest?cb=20130920231337'
+        },
+        {
+          _id: 3,
+          name: 'Wirt',
+          occupation: 'adventurer',
+          avatar: 'http://66.media.tumblr.com/5ea59634756e3d7c162da2ef80655a39/tumblr_nvasf1WvQ61ufbniio1_400.jpg'
+        },
+        {
+          _id: 4,
+          name: 'Michael Myers',
+          occupation: 'Loving little brother',
+          avatar: 'http://vignette2.wikia.nocookie.net/villains/images/e/e3/MMH.jpg/revision/latest?cb=20150810215746'
+        },
+        {
+          _id: 5,
+          name: 'Dana Scully',
+          occupation: 'FBI Agent',
+          avatar: 'https://pbs.twimg.com/profile_images/718881904834056192/WnMTb__R.jpg'
+        }
+      ]
+    };
+  }
+
+  render() {
+    return (
+      <ul className="contact-list">
+        {this.state.contacts.map(contact => {
+          return (
+            <Contact
```

Comment:
One of our classmates had told me about using the spread operator operator as an option for assigning props to the Contact component like this: <Contact {...contact} key={contact._id} />

I thought it was really cool that we can do that and thought I'd share. However I looked at the Facebook React docs and they did advise to use the spread syntax sparingly because it makes it easy to pass in irrelevant props that don't get used. https://facebook.github.io/react/docs/jsx-in-depth.html#spread-attributes

So yeah in this case the spread operator may be helpful since we use all the props but may not be the best option in all scenarios where our state objects have more properties that we don't use.